### DISCUSSION TITLE
Account Suspended

### DISCUSSION THREAD

**Discussion by mferitsimsek** [community/community#127962] [Other Feature Feedback, Questions, & Ideas]
### Select Topic Area

General

### Body

![Ekran görüntüsü 2024-06-10 084403](https://github.com/community/community/assets/172009074/69e33a23-0588-45aa-af4c-1e3bf43c3247)

I opened a support ticket ( #2817983 ) about **11 days ago** with the message “Thank you for contacting GitHub Support. ” I have not received any response other than the first reply. 

My account was banned without being informed about what caused the violation on my account. If I had been informed, I could have eliminated this problem.

I am actively developing software. Not being able to access my Github account causes huge financial losses.

Please support team, I want you to find a solution to my problem as soon as possible.

Thank you. Sincerely yours.

Github Profile : github.com/cptmfs

**Comment by vishal11u** [comment]
Wait for 2days it gives yu mail for reactivation

**Reply by mferitsimsek** [reply]
Today is the 13th day. Do you know anything about estimated response times? Why did you say 2 days?

**Reply by vishal11u** [reply]
give me my answer is right then I will give you proper solution of your problem

**Reply by mferitsimsek** [reply]
I'm not sure I understand. Do you want me to mark as answer (your answer) ?

**Reply by vishal11u** [reply]
yes

**Reply by mferitsimsek** [reply]
I am waiting for your reply

**Reply by vishal11u** [reply]
It's frustrating to experience delays, especially when your work is significantly impacted. Here are some steps you can take to escalate the issue and potentially speed up the resolution process:

### 1. **Follow Up on Your Existing Support Ticket**
Since you haven't received a response beyond the initial acknowledgement, follow up on your existing support ticket to remind them of the urgency. You can reply to the email thread or update the ticket directly through the support portal if available. Here's a template you can use:

```plaintext
Subject: Follow-Up on Ticket #2817983 - Urgent Account Ban Issue

Dear GitHub Support Team,

I am writing to follow up on my support ticket (#2817983), which I submitted 11 days ago. My account was banned without prior notification or explanation. This ban has significantly impacted my ability to develop software, resulting in substantial financial losses.

I kindly request an update on the status of my ticket. If there is any additional information or action required from my side to resolve this issue promptly, please let me know.

Your prompt attention to this matter would be greatly appreciated.

Thank you.

Sincerely,
[Your Name]
GitHub Profile: github.com/cptmfs
```

### 2. **Reach Out via Social Media**
GitHub is often responsive on social media platforms like Twitter. Consider tweeting at GitHub Support (@GitHubHelp) with a brief summary of your issue and your ticket number. Here's an example tweet:

```plaintext
@GitHubHelp My account has been banned without explanation, and my support ticket (#2817983) has been pending for 11 days. This is causing significant financial loss. Can you please assist urgently? Thank you.
```

### 3. **Check for Updates in the Support Portal**
If GitHub has a support portal or dashboard, log in to check for any updates or additional comments from the support team that might not have reached your email.

### 4. **Contact a GitHub Community Moderator**
If you're part of any GitHub community forums or groups, you might find moderators or community managers who can help escalate your issue internally.

### 5. **Review GitHub's Terms of Service and Community Guidelines**
Ensure you understand GitHub's terms and guidelines. Sometimes, familiarizing yourself with these documents can provide insights into potential reasons for the ban and how to address them in your communications with support.

### 6. **Consider Alternative Contact Methods**
If GitHub offers any alternative contact methods (such as a phone number for urgent support), use them to explain your situation and seek immediate assistance.

### 7. **Document All Communications**
Keep a record of all your communications with GitHub support, including emails, tweets, and any other correspondence. This documentation can be helpful if you need to escalate the issue further.

### 8. **Explore Temporary Alternatives**
While waiting for your GitHub account issue to be resolved, consider using alternative version control and repository hosting services like GitLab or Bitbucket to minimize disruption to your development work.

By taking these steps, you can increase the likelihood of getting a timely resolution to your account issue.

**Reply by mferitsimsek** [reply]
Thanks for this comprehensive response

**Comment by ebndev** [comment]
Hi @mferitsimsek,

This community is managed by GitHub employees, but it is not a support forum. You have done everything necessary on your end by opening a ticket on our [Support page](https://support.github.com/contact).

This is a public forum, and the community is unable to assist with nor escalate your ticket, but we can assure you it is in the right place. It will be answered in the order it was received based on our team's resources and the amount of tickets ahead of yours.

As no other Community Discussions team members will be able to provide additional help with account related questions I am going to close this discussion. We appreciate your understanding!
